### PR TITLE
Markets tech life arts

### PR DESCRIPTION
**Pull request by aintgoin2goa**
These are the last of the easy sections.

Left is the Video scroller and the US Election.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,12 @@
...
-    "preact": "^4.8.0",
-    "preact-compat": "1.8.3",
+    "preact": "^6.0.2",
+    "preact-compat": "3.4.2",
     "proptypes": "^0.14.3",
-    "react": "^0.14.6",
-    "react-dom": "^0.14.6"
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "redux": "^3.5.2"
```
Comment (i-like-robots) [inline package.json]: what's using redux?

File: server/sections/life-arts/view.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div id="life-and-arts-low" class="js-track-scroll-event" data-o-grid-colspan="12">
+	<section class="o-grid-row section--life-and-arts" data-trackable="life-and-arts-low">
```
Comment (i-like-robots) [inline server/sections/life-arts/view.html]: I don't think the `o-grid-row` does anything

File: server/sections/life-arts/view.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+		<div data-o-grid-colspan="12" class="section__column section__column--meta">
```
Comment (i-like-robots) [inline server/sections/life-arts/view.html]: because this is just full width anyway

File: server/sections/life-arts/view.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+			<div class="section-meta">
+				<h2 class="section-meta__title"><a href="{{url}}" class="section-meta__link" data-trackable="section-title">{{title}}</a></h2>
+			</div>
+		</div>
+		<div id="life-and-arts-low-section-content" data-o-grid-colspan="12" class="section__column section__column--content">
```
Comment (i-like-robots) [inline server/sections/life-arts/view.html]: and so is this

File: server/sections/markets/fullwidth.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<div id="markets" class="js-track-scroll-event" data-o-grid-colspan="12">
+	<section class="o-grid-row section--markets" data-trackable="markets">
```
Comment (i-like-robots) [inline server/sections/markets/fullwidth.html]: same as above for this template

File: server/sections/markets/halfwidth.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<div id="markets" class="js-track-scroll-event" data-o-grid-colspan="12 M6">
+	<section class="o-grid-row section--markets" data-trackable="markets">
+		<div data-o-grid-colspan="12" class="section__column section__column--meta">
```
Comment (i-like-robots) [inline server/sections/markets/halfwidth.html]: and this =]

File: package.json (hunk 1 above)
Comment (aintgoin2goa) [inline package.json]: Dunno - this is from a merge - I didn't add it

Comment (i-like-robots) [inline package.json]: I think I took it away when babel was updated for node 6

File: server/sections/markets/halfwidth.html (hunk 6 above)
Comment (aintgoin2goa) [inline server/sections/markets/halfwidth.html]: Yeah, it looks like these are needed for some reason, don't want to touch it now

Comment (i-like-robots) [inline server/sections/markets/halfwidth.html]: makes sense, np

**Pull request merged**